proto/viam/app/v1/app.proto

```diff
@@ -4,17 +4,71 @@ package viam.app.v1;
 
 import "google/protobuf/struct.proto";
 import "google/protobuf/timestamp.proto";
-import "tagger/v1/tagger.proto";
 
 option go_package = "go.viam.com/api/app/v1";
 
 service AppService {
-  rpc CreateLocation(CreateLocationRequest) returns (CreateLocationResponse);
+  // Organization
+  //
+
+  // Create an organization
+  rpc CreateOrganization(CreateOrganizationRequest) returns (CreateOrganizationResponse);
 
+  // List all organizations 
   rpc ListOrganizations(ListOrganizationsRequest) returns (ListOrganizationsResponse);
 
+  // Get an organization
+  rpc GetOrganization(GetOrganizationRequest) returns (GetOrganizationResponse);
+  
+  // Update an organization
+  rpc UpdateOrganization(UpdateOrganizationRequest) returns (UpdateOrganizationResponse);
+
+  // Delete an organization
+  rpc DeleteOrganization(DeleteOrganizationRequest) returns (DeleteOrganizationResponse);
+
+  // List all members of an organization
+  rpc ListOrganizationMembers(ListOrganizationMembersRequest) returns (ListOrganizationMembersResponse);
+
+  // Create an organization invite to an organization
+  rpc CreateOrganizationInvite(CreateOrganizationInviteRequest) returns (CreateOrganizationInviteResponse);
+  
+  // Delete an organization member from an organization
+  rpc DeleteOrganizationMember(DeleteOrganizationMemberRequest) returns (DeleteOrganizationMemberResponse);
+
+  // Delete an organization invite
+  rpc DeleteOrganizationInvite(DeleteOrganizationInviteRequest) returns (DeleteOrganizationInviteResponse);
+
+  // Location
+  //
+
+  // Create a location
+  rpc CreateLocation(CreateLocationRequest) returns (CreateLocationResponse);
+
+  // List all locations
   rpc ListLocations(ListLocationsRequest) returns (ListLocationsResponse);
 
+  // Get a location
+  rpc GetLocation(GetLocationRequest) returns (GetLocationResponse);
+
+  // Update a location
+  rpc UpdateLocation(UpdateLocationRequest) returns (UpdateLocationResponse);
+
+  // Delete a location
+  rpc DeleteLocation(DeleteLocationRequest) returns (DeleteLocationResponse);
+
+  // Update a location's parent organization
+  rpc UpdateLocationParent(UpdateLocationParentRequest) returns (UpdateLocationParentResponse);
```

Comment:
We could instead have a `parent_location_id` field on `UpdateLocationRequest` instead of having `UpdateLocationParent`